pkg/kubelet/kubelet.go

```diff
@@ -697,8 +702,9 @@ func makeBinds(container *api.Container, podVolumes volumeMap) (binds []string)
 func (kl *Kubelet) GenerateRunContainerOptions(pod *api.Pod, container *api.Container, netMode, ipcMode string) (*kubecontainer.RunContainerOptions, error) {
 	var err error
 	opts := &kubecontainer.RunContainerOptions{
-		NetMode: netMode,
-		IpcMode: ipcMode,
+		NetMode:    netMode,
+		IpcMode:    ipcMode,
+		CgroupRoot: kl.cgroupRoot,
```

Comment:
If we are introducing this, should we also introduce the pod-level cgroup?

/kubernetes/pod-uid/container-id

And do we want to distinguish pods that asked for limits from pods that didnt?  And is that CPU, memory, or the tuple of both?

/kubernetes/t1/pod-uid/container-id

Let's also put thought into how to future-proof this.  Changing this schema has been an unending source of pain internally.

Also, let's carefully consider how important this is before 1.0.  It's fun, but we want to be careful about what we undertake in the near-term.